components/gitpod-protocol/data/gitpod-schema.json

```diff
@@ -71,11 +71,11 @@
                     },
                     "before": {
                         "type": "string",
-                        "description": "A shell command to run before `init` and the main `command`. This command is executed on every start and is expected to terminate. If it fails, the following commands will not be executed."
+                        "description": "A shell command to run before `init` and the main `command`."
                     },
                     "init": {
                         "type": "string",
-                        "description": "A shell command to run between `before` and the main `command`. This command is executed only on after initializing a workspace with a fresh clone, but not on restarts and snapshots. This command is expected to terminate. If it fails, the `command` property will not be executed."
```

Comment:
Same here: Either leave/update the lifecycle comment, or point to the docs directly: https://www.gitpod.io/docs/configure/workspaces/tasks#execution-order

Reply:
I felt it wasn't fully up-to-date or clear covering all the cases (see [previous comment](https://github.com/gitpod-io/gitpod/pull/16184/files#r1098796318)).

Happy to put the docs link instead, WDYT?